assertj-core/src/main/java/org/assertj/core/internal/Strings.java

```diff
@@ -469,6 +475,11 @@ private static String normalizeWhitespace(CharSequence toNormalize) {
     return result.toString().trim();
   }
 
+  private static boolean isBreakingSpace(Character character) {
+    return stream(BREAKING_SPACES)
+      .anyMatch(character::equals);
```

Comment:
I could have written:

`java.util.Arrays.asList(BREAKING_SPACES).contains(character);` which is equivalent but I don't think that its more readable. If you think that's better, I can change.

Reply:
Superseded by the usage of a static `Set`.